src/components/ProfilePage.tsx

```diff
@@ -131,98 +146,189 @@ export const ProfilePage = (props: ProfilePageProps) => {
             <input
                 type={'file'}
                 ref={hiddenFileInput}
-                onChange={uploadHandler}
-                style={{display: 'none'}}
+                onChange={onSelectFile}
+                style={{ display: 'none' }}
             />
-            <Button style={{marginTop:'16px'}} variant='secondary' onClick={clickHandler}>Upload File</Button>
+            <Button
+                className='upload-btn'
+                variant='secondary' onClick={clickHandler}>
+                Upload File
+            </Button>
         </>
     )
 
     const decoratedProfilePic = (
         <>
-            {verified ? <div className='verified-img-container'>
-                <img className='verified-border' src={VerifiedBorder}/>
-                <img className='verified-img' src={profilePicUrl}/>
-                {editing && uploadImg}
-            </div> :
-            <div className='non-verified-profile-pic'>
-                <img className='non-verified-img' src={profilePicUrl}/>
-                {editing && uploadImg}
-            </div>
+            {
+                verified ?
+                    <div className='verified-img-container'>
+                        <img className='verified-border' src={VerifiedBorder} />
+                        <img className='verified-img' src={profilePicUrl} />
+                        {editing && uploadImg}
+                    </div>
+                    :
+                    <div className='non-verified-profile-pic'>
+                        <img className='non-verified-img' src={profilePicUrl} />
+                        {editing && uploadImg}
```

Comment:
As above (use whatever name you decide is most appropriate)
```suggestion
                        {editing && <UploadImage/>}
```